wolfssl/ssl.h

```diff
@@ -4553,7 +4557,7 @@ WOLFSSL_API WOLFSSL_X509_NAME_ENTRY *wolfSSL_X509_NAME_get_entry(WOLFSSL_X509_NA
 WOLFSSL_API void wolfSSL_X509_NAME_ENTRY_free(WOLFSSL_X509_NAME_ENTRY* ne);
 WOLFSSL_API WOLFSSL_X509_NAME_ENTRY* wolfSSL_X509_NAME_ENTRY_new(void);
 WOLFSSL_API void wolfSSL_X509_NAME_free(WOLFSSL_X509_NAME* name);
-WOLFSSL_API char wolfSSL_CTX_use_certificate(WOLFSSL_CTX* ctx, WOLFSSL_X509* x);
+WOLFSSL_API int wolfSSL_CTX_use_certificate(WOLFSSL_CTX* ctx, WOLFSSL_X509* x);
```

Comment:
Was the change from char to int necessary or a clean up to make it match other functions? This will likely break ABI and looks like it's been a char a long time. Am on the fence if we should just change it anyway like what's done in this PR....

Reply:
To clarify breaking API backwards compatibility, in that we need to update SONAME next release, not breaking WOLFSSL_ABI.

Reply:
OpenSSL has the API return an int.
The returned values were and are WOLFSSL_SUCCESS and WOLFSSL_FAILURE so using int won't affect the return values.
Saw it as fair game as it didn't have WOLFSSL_ABI.